src/main/java/duke/processors/Ui.java

```diff
@@ -0,0 +1,52 @@
+package duke.processors;
+
+import duke.exception.DukeEmptyCommandException;
+import java.util.Scanner;
+public class Ui {
+    private String separation = "_______________________________________________";
```

Comment:
Perhaps this could be a final String? 🤔 
Maybe could also capitalise the constant name? 

For example:
```java
private final String SEPARATION = "_______________________________________________";
```